### PR TITLE
Release 1.6.0

### PR DESCRIPTION
**Pull request by emmas-yoti**
### Added
 * Document Details, available through `Yoti::Profile#document_details`
 * Access to lists of age verifications through `Yoti::Profile#age_verifications`, `Yoti::Profile#find_age_over_verification(age)` and `Yoti::Profile#find_age_under_verification(age)`
 * Dynamic Share support through `Yoti::DynamicSharingService`
    * `DynamicScenario` to create a specification for a dynamic share
    * `DynamicPolicy` to create a specification for wanted attributes and authorisation types
    * `WantedAttribute` to create a specification for an individual attribute
    * `create_share_url` to request a share QR for a Dynamic Scenario
    * `SourceConstraint` to restrict anchors used to satisfy a dynamic share
    * `WantedAnchor` to define the specification for an anchor
 * Third Party Attribute support
   * `extra_data` to  `ActivityDetails`
   * `ThirdpartyAttributeExtension` to issue third party attributes when Yoti Hub is set up with support

### Changed
 * `SignedRequest` no longer sets usecase specific headers (such as X-Yoti-Auth-Key and 'Content-Type') and only sets SDK Name and Version, unless used through the Yoti Client

### Deprecated
- `Yoti::ActivityDetails#age_verified`
- `Yoti::AgeProcessor`

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - package_manager: "ruby:bundler"
+    directory: "/"
+    update_schedule: "live"
+    target_branch: "development"
+
+    default_reviewers:
+      - "echarrod"
+      - "davidgrayston"
+      - "emmas-yoti"
+      - "MrBurtyyy"
```
Comment (davidgrayston) [inline .dependabot/config.yml]: Do we need to exclude this file in in https://github.com/getyoti/yoti-ruby-sdk/blob/master/yoti.gemspec#L21 ?

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,9 @@
+sonar.projectKey = yoti-web-sdk:ruby
+sonar.projectName = ruby-sdk
+sonar.projectVersion = 1.5.0
+sonar.language=ruby
+sonar.exclusions = **/protobuf/**.rb, coverage/**, spec/sample-data/**
+sonar.links.scm = https://github.com/getyoti/yoti-ruby-sdk
+sonar.sources = ./lib
+sonar.ruby.coverage.reportPaths = coverage/.resultset.json
+sonar.verbose = true
```
Comment (davidgrayston) [inline sonar-project.properties]: Should we also exclude this in the gem spec?

File: sonar-project.properties [hunk 3]
```diff
@@ -0,0 +1,9 @@
+sonar.projectKey = yoti-web-sdk:ruby
+sonar.projectName = ruby-sdk
+sonar.projectVersion = 1.5.0
```
Comment (davidgrayston) [inline sonar-project.properties]: Should this be changed to `1.6.0`?

File: sonar-project.properties (hunk 2 above)
Comment (emmas-yoti) [reply sonar-project.properties]: > Just the Sonar version
> 
> _(the gemspec exclusions could be done in a follow up ticket - we may want to exclude other files like `.travis.yml`)_

I've added those exclusions.

Comment (davidgrayston) [reply sonar-project.properties]: Tested locally and excludes as expected 👍

**Pull request merged**